assets/queries/dockerCompose/check_cap_add_cap_drop/metadata.json

```
@@ -0,0 +1,10 @@
+{
+    "id": "ce76b7d0-9e77-464d-b86f-c5c48e03e22d",
+    "queryName": "Check Cap Add Cap Drop",
```

I suggest something like:

```suggestion
    "queryName": "Capabilities Unrestricted",
```

If you change the `queryName`, please also update the query folder name